x-pack/plugin/esql/qa/testFixtures/src/main/resources/docs.csv-spec

```diff
@@ -384,6 +384,26 @@ Bernatsky      |true
 // end::startsWith-result[]
 ;
 
+docsEndsWith
```

Comment:
I just put these into string.csv-spec usually. `docs-csv-spec` I think was a mistake. there are too many of these in the docs to put them all in one place!

Reply:
fair point, if we continue like this the docs spec will blow up. I've moved them to the strings spec as suggested. Thx!